setup.py

```diff
@@ -1,10 +1,10 @@
 #! /usr/bin/env python
 
-from setuptools import setup, Extension
+from setuptools import setup
 import importlib
-import os
+from importlib import util
```

Comment:
Do we use `util` somewhere?

Reply:
For some reason I got local ``Cannot find reference 'util' in '__init__.pyi' `` when just using ``import importlib``.  Adding that line made it go away. I think I'll  just remove the first ``import importlib`` and use calls directly to ``util``